src/pipeline_v3/tests/integration/test_e2e_integration.py

```
@@ -127,6 +144,8 @@
                         assert model in markdown, f"Expected model {model} not found"
 
             except Exception as e:
+                print(f"Exception during processing: {type(e).__name__}: {e}")
+                print(f"Result before exception: {result if 'result' in locals() else 'No result yet'}")
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)